src/som/interpreter/actors/EventualMessage.java

```diff
@@ -323,24 +331,40 @@ public static EventualMessage getCurrentExecutingMessage() {
 
   /**
    * Indicates that execution should stop and yield to the debugger,
-   * before the computed value is used to resolve the promise.
+   * before the message is processed.
+   */
+  public boolean haltOnReceive() {
+    return haltOnReceive;
```

Comment:
The link isn't really a style guide.
Either way, `isEnabledHaltOnReceive()` is much longer, and not much more grammatical to my ears.

If you answer my question why it is confusing, I am considering to name it `getHaltOnReceive()`.